docs/csharp/cs-dev-kit-faq.md

```diff
@@ -258,3 +258,13 @@ To resolve this issue, clear out old folders from within the `obj` folder or cle
 ### I am not getting whole line completions
 
 Whole line completions are disabled when the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension is enabled to allow you to take advantage of the more advanced [AI completion](/docs/editor/github-copilot.md) capabilities. You can verify that Copilot is enabled by checking if the Copilot logo is present in the lower right corner of VS Code.
+
+## Hot Reload
```

Comment:
>## [](http://example.com/codeflow?start=0&length=2)

Should we add something like "what types of edits are supported?" and link to [https://learn.microsoft.com/en-us/visualstudio/debugger/supported-code-changes-csharp?view=vs-2022](https://learn.microsoft.com/en-us/visualstudio/debugger/supported-code-changes-csharp?view=vs-2022)